src/main/scala/com/gd/twitteranalytics/TwitterReportingJob.scala

```diff
@@ -45,11 +47,7 @@ object TwitterReportingJob {
       System.exit(1)
     }
   }
-
   def setSparkSession:SparkSession = {
-      SparkSession.builder()
-                  .appName("TwitterKPIReport")
-                  .getOrCreate()
+      SparkSession.builder().appName("TwitterKPIReport").getOrCreate()
```

Comment:
Again... There are going to be many different reports, many different KPIs. This fact should be taken into account right away, i.e. the design and naming should accommodate future extension (e.g. adding a new  report should not affect existing ones, lookup/search of a particular report should be straightforward)

Reply:
Fixed.